src/main.cc

```diff
@@ -142,8 +142,8 @@ int main(int argc, char *argv[])
     // Allow for command line override of renderer
     for (int i = 0; i < argc; i++) {
         const QString arg(argv[i]);
-        if (arg == QStringLiteral("-angle")) {
-            QCoreApplication::setAttribute(Qt::AA_UseOpenGLES);
+        if (arg == QStringLiteral("-desktop")) {
```

Comment:
Isn't this the default. Same as specifying nothing on the command line?

Reply:
It didn't work until I added that so I don't necessarily think so? I was getting a black screen without that.

Reply:
ok, fine